scipy/special/__init__.py

```diff
@@ -548,6 +548,7 @@
    roots_sh_chebyt   -- Gauss-Chebyshev (first kind, shifted) quadrature.
    roots_sh_chebyu   -- Gauss-Chebyshev (second kind, shifted) quadrature.
    roots_sh_jacobi   -- Gauss-Jacobi (shifted) quadrature.
+   roots_lebedev     -- Lebedev quadrature.
```

Comment:
Just want to point out that I'll leave it to others to assess whether it is appropriate to put this under the heading of "roots and quadrature weights for orthogonal polynomials:".